### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the content of "Sign in with a username and password instead" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1495]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5. In" Azure Login" Enter the "email or phone number" in the edit box and select Next button.
6. Tab to and select "Sign in with a user name and password instead" link.
7. Enter the required password and select "sign in" button.
8. Observe Jaws and NVDA behaviour.

**Actual Result:**
Jaws and NVDA are not announcing the content present in "Sign in with a username and password instead" dialog 

**Expected Result:**
Jaws and NVDA should announce the content of the "Sign in with a username and password instead" dialog 

Jaws and NVDA should announce as
Azure Login Dialog
tab
Sign in with PIN or smart card instead button
tab
Sign in with a username and password instead

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![c67f4671](https://user-images.githubusercontent.com/22956130/36136472-a3c1b4ee-10b6-11e8-92a5-242c27fe7113.jpg)
![azure login dialog-signin1](https://user-images.githubusercontent.com/22956130/36136474-a797371a-10b6-11e8-855d-1fdd5f0f1358.jpg)

**Comment by Eskibear** [comment]
duplicate with #1496

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749947
